code/src/components/SelectForm.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react'
+
+const SelectForm = ({input, setInput}) => {
+    return (
+        <div className="select-form-div">
```

Comment:
It doesn't seem like you use `className="select-form-div"` in your stylesheet. Do you think it's better if you remove it, if you are not using it?